rock_paper_scissors.js

```diff
@@ -0,0 +1,43 @@
+var userChoice = prompt("Wanna Play a game of rock, paper or scissors?");
+document.write("You Selected: " + userChoice);
+document.write("<br>");
+var computerChoice = Math.random();
+	if (computerChoice <= 0.33) {
+		computerChoice = "rock";
+	} else if(computerChoice >= 0.34 && computerChoice <=0.67) {
+		computerChoice = "paper";
+	} else{
+	computerChoice = "scissors";
+	} 
+document.write("Computer Selected: " + computerChoice);
+document.write("<br>");
+var compare = function (choice1, choice2) {
+	if (choice1 === choice2) {
+		return "It's a Stalemate - Try again!";
+	} else if(choice1 === "rock") {
+		if(choice2 === "scissors") {
+			return "Rock wins";

```

Comment:
If you look, you are always returning `rock/scissors/paper wins`. You can return only the "winner" e.g. Rock and put "wins to the message" :)